modules/Bio/EnsEMBL/Production/Pipeline/FileDump/README.txt

```diff
@@ -6,39 +6,69 @@ The structure of the FTP directory is show below.
 <species> is the scientific name of a species.
 <assembly> is the assembly accession, e.g. GCA_001458135.1.
 <geneset> is the date that the geneset was loaded by Ensembl.
+<species.annotation_source> is the source of annotation used. See below.
 Square brackets indicate directories or files that are not available
 for all species.
 
 species
   |-- <species>
       |-- <assembly>
-          |-- geneset
-              |-- <geneset>
+          |-- <species.annotation_source>
+              |-- geneset
+                  |-- <geneset>
+                      |-- md5sum.txt
+                      |-- README.txt
+                      |-- <species>-<assembly>-<geneset>-cdna.fa.gz
+                      |-- <species>-<assembly>-<geneset>-cds.fa.gz
+                      |-- <species>-<assembly>-<geneset>-genes.embl.gz
+                      |-- <species>-<assembly>-<geneset>-genes.gtf.gz
+                      |-- <species>-<assembly>-<geneset>-genes.gff3.gz
+                      |-- <species>-<assembly>-<geneset>-pep.fa.gz
+                      |-- <species>-<assembly>-<geneset>-xref.tsv.gz
+              |-- genome
                   |-- md5sum.txt
                   |-- README.txt
-                  |-- <species>-<assembly>-<geneset>-cdna.fa.gz
-                  |-- <species>-<assembly>-<geneset>-cds.fa.gz
-                  |-- <species>-<assembly>-<geneset>-genes.embl.gz
-                  |-- <species>-<assembly>-<geneset>-genes.gtf.gz
-                  |-- <species>-<assembly>-<geneset>-genes.gff3.gz
-                  |-- <species>-<assembly>-<geneset>-pep.fa.gz
-                  |-- <species>-<assembly>-<geneset>-xref.tsv.gz
-          |-- genome
-              |-- md5sum.txt
-              |-- README.txt
-              |-- [assembly_mapping]
-                  |-- *.chain.gz
-              |-- [<species>-<assembly>-chromosomes.tsv.gz]
-              |-- <species>-<assembly>-hardmasked.fa.gz
-              |-- <species>-<assembly>-softmasked.fa.gz
-              |-- <species>-<assembly>-unmasked.fa.gz
-          |-- [rnaseq]
-              |-- md5sum.txt
-              |-- README.txt
-              |-- *.bam
-              |-- [*.bam.bai]
-              |-- *.bam.bw
-              |-- [*.bam.csi]
+                  |-- [assembly_mapping]
+                      |-- *.chain.gz
+                  |-- [<species>-<assembly>-chromosomes.tsv.gz]
+                  |-- <species>-<assembly>-hardmasked.fa.gz
+                  |-- <species>-<assembly>-softmasked.fa.gz
+                  |-- <species>-<assembly>-unmasked.fa.gz
+              |-- [homology]
+                  |-- <species>-<assembly>-<geneset>-homology.tsv.gz
+              |-- [rnaseq]
+                  |-- md5sum.txt
+                  |-- README.txt
+                  |-- *.bam
+                  |-- [*.bam.bai]
```

Comment:
This is for rapid right? So we could remove the bam file info now

Reply:
Sure! Thanks for spotting it! @leannehaggerty  otherwise is it all good?